deploy/olm-catalog/ibm-block-csi-operator-community/1.4.0/ibm-block-csi-operator.v1.4.0.clusterserviceversion.yaml

```diff
@@ -4,7 +4,7 @@ metadata:
   name: ibm-block-csi-operator.v1.4.0
   namespace: placeholder
   annotations:
-    capabilities: "Basic Install"
+    capabilities: "Seamless Upgrades"
```

Comment:
that is [not true](https://github.com/IBM/ibm-block-csi-operator/pull/61)

Reply:
will be lint override on internal files

Reply:
updated - back to original value

Reply:
still doesn't look like the original value

Reply:
@oriyarde , pls check